dali/operators/reader/numpy_reader_op.cc

```diff
@@ -122,6 +122,14 @@ This argument is mutually exclusive with ``files``.)", nullptr)
 If ``file_root`` is provided, the paths are treated as being relative to it.
 
 This argument is mutually exclusive with ``file_list``.)", nullptr)
+  .AddOptionalArg("register_buffers", R"code(
+.. warning::
+this argument is temporarily disabled and left for the backward compatibility.
+It will be reenabled in the future releases.
```

Comment:
```suggestion
  This argument is temporarily disabled and left for the backward compatibility.
  It will be reenabled in the future releases.
```
You can test this using http://rst.ninjs.org/.